docs/src/pages/developers/mappings.mdx

```diff
@@ -11,7 +11,7 @@ title: Importing structured data
 
 Aleph is commonly used to import data tables \(like a companies registry, list of persons of interest, or a set of government contracts\) that a user wants to search and browse.
 
-It does this by mapping tabular source data to the [Follow the Money](https://docs.alephdata.org/developers/followthemoney) \(FtM\) data model. Aleph defines entities \(such as `People`, `Companies`, `Assets` or `Court Cases`\), and the relationships between them \(such as `Family`relations, or business interests – `Ownership` or `Directorship`, or other links like `Sanctions`, `Payments` \).
+It does this by mapping tabular source data to the [Follow the Money](/developers/followthemoney) \(FtM\) data model. Aleph defines entities \(such as `People`, `Companies`, `Assets` or `Court Cases`\), and the relationships between them \(such as `Family`relations, or business interests – `Ownership` or `Directorship`, or other links like `Sanctions`, `Payments` \).
```

Comment:
@tillprochaska This is ok without the domain? I notice this a few times in the changes you've made, I assume that it is ok, but wanted to check and be sure

Reply:
Argh, but I’ve actually made one mistake and also used a relative URL in a Markdown file for the page at https://aleph.occrp.org/pages/faq where this doesn’t work. I’ll fix this shortly